pkg/resources/gce.go

```diff
@@ -735,24 +734,25 @@ func (d *clusterDiscoveryGCE) matchesClusterName(name string) bool {
 }
 
 func (d *clusterDiscoveryGCE) listGCEDNSZone() ([]*tracker.Resource, error) {
-	if dns.IsGossipHostname(d.clusterName) {
-		return nil, nil
-	}
-
-	zone, err := d.findDNSZone()
-	if err != nil {
-		return nil, err
-	}
-
-	return []*tracker.Resource{
-		{
-			Name:    zone.Name(),
-			ID:      zone.Name(),
-			Type:    "DNS Zone",
-			Deleter: d.deleteDNSZone,
-			Obj:     zone,
-		},
-	}, nil
+	// We never delete the hosted zone, because it is usually shared and we don't create it
+	return nil, nil
+	//if dns.IsGossipHostname(d.clusterName) {
```

Comment:
Can we just remove the code?

Reply:
I want to keep it so that when we merge the Shared-resource PR we can reintroduce it.

Will add a comment.